modules/gitops-argocd-cli-global-options.adoc

```diff
@@ -0,0 +1,122 @@
+// Module is included in the following assemblies:
+//
+// * gitops_cli_argocd/argocd-gitops-cli-reference.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="gitops-argocd-cli-global-options_{context}"]
+= Global options
+
+Global options are applicable to all the subcommands of `argocd`.
+
+.Global options
+[options="header",cols="1,1,1"]
+|===
+|Option |Argument type |Description
+
+|`--auth-token`
+|`string`
+|Authentication token.
+
+|`--client-crt`
+|`string`
+|Client certificate file.
+
+|`--client-crt-key`
+|`string`
+|Client certificate key file.
+
+|`--config`
+|`string`
+|Path to Argo CD configuration file. Defaults to `/home/user/.config/argocd/config` path.
+
+|`--controller-name`
+|`string`
+|Name of the Argo CD Application Controller component with `argocd-application-controller` as the default label. 
+
+If the label of this component's name differs from the default, for example, when you are installing it through the Helm chart, set either the `--controller-name` option or the `ARGOCD_APPLICATION_CONTROLLER_NAME` environment variable to specify the name of the component. 
+
+|`--core`
+|N/A
+|If set to `true`, the CLI communicates directly with the Kubernetes API server instead of using the Argo CD API server.
+
+|`--grpc-web`
+|N/A
+|Enable the gRPC-Web protocol for the Argo CD server, for example, if the server is behind a proxy that does not support the HTTP/2 protocol.
+
+|`--grpc-web-root-path`
+|`string`
+|Enable the gRPC-Web protocol for the Argo CD server, for example, if the server is behind a proxy that does not support the HTTP/2 protocol. Set web root.
```

Comment:
Would this enable the gRPC-Web protocol if the `--grpc-web` flag is not used? It sounds like this is the case based on the wording here.

Reply:
@anandf Please clarify and confirm.

Reply:
yes, if `--grpc-web-root-path` is used then gRPC-Web protocol is enabled if `--grpc-web` flag is not used.